src/docs/product/performance/transaction-summary.mdx

```diff
@@ -101,6 +100,17 @@ The **Tags** tab displays a list of suspect tag keys that often correspond to sl
 
 ![Example of Tags Tab](./tags-tab.png)
 
+## Anomalies
+
+<Note>
+
+This feature is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony.
+If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.
+
+</Note>
+
+The time series graph in the **Anomalies** tab shows your event throughput over time. Our anomaly detection highlights areas that have gone above or below the expected throughput for that time of day, week, or other period, based on your historical data. The different colors for anomalies represent the level of confidence we have that that area is anomalous.
```

Comment:
should we say "transaction throughput" rather than "event throughput" here for the time being?

Reply:
I'm either/or on this usually since our users might not really conceive of 'transactions' as events, whereas events and epm always applies. We actually mix these a lot in the product, so I don't know that we've figured out which we prefer either.

Reply:
fair enough!